specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Credential.json

```diff
@@ -61,6 +61,7 @@
       "properties": {
         "servicePrincipalId": {
           "type": "object",
+          "x-ms-format": "dfe-string",
```

Comment:
Can you please fill out the PR description survey form in this PR? and also, share as why x-ms-format as dfe-string is being added and what does it mean?